### DISCUSSION TITLE
Welcome to tskit Discussions!

### DISCUSSION THREAD

**Discussion by benjeffery** [tskit-dev/tskit#1093] [General]
## 👋 Welcome!
  Github have added discussions - we're just experimenting with them at the moment. They could be good for user support? Or maybe feature discussions that are too un-concrete for an issue? 

Please use as you see fit, and we'll see what sticks!

**Comment by benjeffery** [comment]
Discussions are not like regular issues, you can reply to a perticular message, making a TREE! A set of discussions is therefore a TREE SEQUENCE! We should sue github for prior art.

Things can be voted for, and marked as answers, which would be good for a Q and A format.

**Comment by agladstein** [comment]
Maybe this solves the question of where do users ask for help?
